diff -Nrup lldpd-0.9.3-orig/src/daemon/lldpd.c lldpd-0.9.3/src/daemon/lldpd.c
--- lldpd-0.9.3-orig/src/daemon/lldpd.c	2016-05-21 01:32:53.000000000 +0800
+++ lldpd-0.9.3/src/daemon/lldpd.c	2016-05-30 14:57:43.544007216 +0800
@@ -1780,8 +1780,7 @@ lldpd_main(int argc, char *argv[], char
 		calloc(1, sizeof(struct lldpd_chassis))) == NULL)
 		fatal("localchassis", NULL);
 	cfg->g_config.c_cap_advertise = 1;
-	lchassis->c_cap_available = LLDP_CAP_BRIDGE | LLDP_CAP_WLAN |
-	    LLDP_CAP_ROUTER | LLDP_CAP_STATION;
+	lchassis->c_cap_available = LLDP_CAP_BRIDGE;
 	cfg->g_config.c_mgmt_advertise = 1;
 	TAILQ_INIT(&lchassis->c_mgmt);
 #ifdef ENABLE_LLDPMED
